File: backjoon/Swift/backjoon_9498.swift
//
//  backjoon_9498.swift
//  algo
//
//  Created by 이재은 on 06/06/2020.
//  Copyright © 2020 이재은. All rights reserved.
//

import Foundation

//backjoon 9498 시험 성적
//
//문제
//시험 점수를 입력받아 90 ~ 100점은 A, 80 ~ 89점은 B, 70 ~ 79점은 C, 60 ~ 69점은 D, 나머지 점수는 F를 출력하는 프로그램을 작성하시오.
//
//입력
//첫째 줄에 시험 점수가 주어진다. 시험 점수는 0보다 크거나 같고, 100보다 작거나 같은 정수이다.
//
//출력
//시험 성적을 출력한다.

let score = Int(readLine()!)!

switch score {
case 90...100:
    print("A")
case 80...89:
    print("B")
case 70...79:
    print("C")
case 60...69:
    print("D")
default:
    print("F")
}

File: backjoon/Python/backjoon_9498.py
##backjoon 9498 시험 성적
##
##문제
##시험 점수를 입력받아 90 ~ 100점은 A, 80 ~ 89점은 B, 70 ~ 79점은 C, 60 ~ 69점은 D, 나머지 점수는 F를 출력하는 프로그램을 작성하시오.
##
##입력
##첫째 줄에 시험 점수가 주어진다. 시험 점수는 0보다 크거나 같고, 100보다 작거나 같은 정수이다.
##
##출력
##시험 성적을 출력한다.

score = int(input())

if score >= 90: print("A")
elif score >= 80: print("B")
elif score >= 70: print("C")
elif score >= 60: print("D")
else: print("F")
